src/SparkTestBase.sol

```diff
@@ -40,22 +45,133 @@ interface IExecutable {
     function execute() external;
 }
 
-abstract contract SparkTestBase is Test {
-    address internal executor;
-    address internal payload;
+abstract contract SpellRunner is Test {
+    using DomainHelpers for Domain;
+    using DomainHelpers for StdChains.Chain;
+
+    struct ChainSpellMetadata{
+      address   payload;
+      IExecutor executor;
+      Domain    domain;
+      /// @notice on mainnet: zero
+      /// on L2s: mainnet address of the bridge that'll include txs in the L2
```

Comment:
`Bridge` is a struct that contains multiple addresses. I would just say this is set for L2s and not set for mainnet.

Reply:
with a91a65b now `ChainSpellMetadata` has a Bridge array, which is empty for mainnet. fixed the docs on 0cb4d55